src/riak_cs_config.erl

```diff
@@ -488,6 +462,7 @@ local_get_block_timeout() ->
 ?TIMEOUT_CONFIG_FUNC(get_index_range_gckeys_timeout).
 ?TIMEOUT_CONFIG_FUNC(get_index_range_gckeys_call_timeout).
 ?TIMEOUT_CONFIG_FUNC(get_index_list_multipart_uploads_timeout).
+?TIMEOUT_CONFIG_FUNC(cluster_id_timeout).
```

Comment:
The configuration item is not introduced by this PR. [1] Changing the name will bring in backward incompatibility.

[1] at tag 2.0: https://github.com/basho/riak_cs/blob/2.0/src/riak_cs_config.erl#L256